docs/tutorials/tutorial-custom-domain-with-istio.md

```diff
@@ -97,21 +105,22 @@ The next snipplet allows Gardener to manage certificates automatically from *[Le
 ```
 
 {{% alert color="info" %}}
-<p>Adjust the snipplets with your parameters (don't forget your email). And please use the mydomain-staging issuer while you are testing and learning. Otherwise, Let's Encrypt will rate limit your frequent requests and you can wait a week until you can continue.</p>
+Adjust the snipplets with your parameters (don't forget your email). And please use the mydomain-staging issuer while you are testing and learning. Otherwise, Let's Encrypt will rate limit your frequent requests and you can wait a week until you can continue.
 {{% /alert %}}
 
 References for [Let's Encrypt](https://letsencrypt.org):
+
 * [Rate limit](https://letsencrypt.org/docs/rate-limits/)
 * [Staging environment](https://letsencrypt.org/docs/staging-environment/)
 * [Challenge Types](https://letsencrypt.org/docs/challenge-types/)
 * [Wildcard Certificates](https://community.letsencrypt.org/t/acme-v2-production-environment-wildcards/55578)
 
 ## Create the Gardener Shoot Cluster
 
-Remember I chose to create the Shoot on GCP, so below is the simplest declarative shoot or cluster order document. Notice that I am referring to the infrastructure credentials with `shoot-operator-gcp` and I combined the above snipplets into the yaml file:
+Remember I chose to create the Shoot on GCP, so below is the simplest declarative shoot or cluster order document. Notice that I am referring to the infrastructure credentials with `shoot-operator-gcp` and I combined the above snippets into the yaml file:
 
 ```yaml
-apiVersion: core.gardener.cloud/v1beta1
+apiVersion: core.gardener.cloud/v1
```

Comment:
Do we want to use `v1beta1` here of `v1`?

Reply:
For Shoots we still have API Version `core.gardener.cloud/v1beta1`, that is correct as it is.